js/articles.js

```diff
@@ -1,34 +1,36 @@
+'use strict';
+
 var ourArticles = [
   {
     title: 'Lushootseed',
     authors: ['Alison Zerbe', 'Aziza Houni', 'Derek Hewitt', 'Elizabeth Kleinschmidt'],
```

Comment:
Not a point ding, but a note. In addition to having the individual authors, you might also want to link to their github pages from your portfolio. Again, not a downgrade just a suggestion.